Doc/library/io.rst

```diff
@@ -168,6 +168,19 @@ High-level Module Interface
    :func:`os.stat`) if possible.
 
 
+.. data:: SEEK_SET
+          SEEK_CUR
+          SEEK_END
+
+   Parameters to the :meth:`~IOBase.seek` method,
+   used to specify the relative position
+   Their values are 0, 1, and 2, respectively.
```

Comment:
Perhaps a bit verbose, but I think it might be useful to spell out what each constant means. Also this allows putting the values in-line.

```suggestion
   Parameters to the :meth:`~IOBase.seek` method,
   used to specify the relative position to seek from.

   :data:`!SEEK_SET`
     Seek from the start of the stream
     (Value: ``0``)
   :data:`!SEEK_CUR`
     Seek from the current stream position
     (Value: ``1``)
   :data:`!SEEK_END`
     Seek from the end of the stream
     (Value: ``2``)

```

Reply:
Thanks; applied but not pushed.